package.json

```diff
@@ -31,29 +35,35 @@
       "no-param-reassign": 0
     }
   },
-  "repository": "github:hydra-newmedia/hapi-sentry",
+  "engines": {
+    "node": ">=8.9.0"
```

Comment:
this should be `>=12.0.0` i think, to mirror the hapi requirement. but do we need that, dunno?

Reply:
:shrug: